data_ns: ''
maneuver_ns: ''
visual_ns: ''

state_machine:
  process_freq: 20.0

  # takeoff
  takeoff_alt_m: 3.0
  reached_dist_th_m: 0.4 # checking have reached to mission destination waypoint
  guidance_dist_th_m: 0.7 # checking have reached to next waypoint during flight mode

  # landing
  landing_threshold_m: 0.5 # for distingush kuam control and px4 control during landing 
  landing_standby_alt_m: 3.0 # waiting for detecting marker during landing state
  landing_duration_s: 120
  standby_dist_th_m: 1.5 # checking have reached to landing standby waypoint during landing state
  virtual_border_max_side_m: 6.0
  virtual_border_angle_0_deg: 20.0
  virtual_border_angle_1_deg: 10.0
  alt_division_m: 10.0

  # flight
  alt_offset_m: 23.156

  # emerg
  battery_th_per: 0.10